backend/inspirehep/records/serializers/bibtex.py

```diff
@@ -136,8 +136,10 @@ def serialize(self, pid, record, links_factory=None):
     def serialize_search(
         self, pid_fetcher, search_result, links=None, item_links_factory=None
     ):
-        records = [hit["_source"] for hit in search_result["hits"]["hits"]]
-        return self.create_bibliography(records)
+        records = [
+            hit["_source"]["_bibtex_display"] for hit in search_result["hits"]["hits"]
+        ]
+        return "\n".join(records)
```

Comment:
maybe add an extra newline here, like for LaTeX?

Reply:
I checked how it was before and we had only one newline for bibtex, should I change it to 2?

Reply:
Yes, I think it would be better to separate the entries by a blank line.

Reply:
Actually, I just tested again locally and the bibtex entry already has a newline at the end so we need to add only one here. I'll switch it back